docs/pages/guides/configuration.mdx

```diff
@@ -24,9 +24,9 @@ The following is an example of the `dataform.json` file for a BigQuery project:
 ```json
 {
   "warehouse": "bigquery",
+  "defaultDatabase": "my-gcp-project-id",
   "defaultSchema": "dataform",
-  "assertionsSchema": "dataform_assertions",
```

Comment:
Actually sorry - can you change the file `content/docs/guides/configuration.md` instead? I'm about to delete this.